libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigator.java

```diff
@@ -0,0 +1,51 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+import android.location.Location;
+
+import com.mapbox.geojson.Point;
+import com.mapbox.navigator.FixLocation;
+import com.mapbox.navigator.NavigationStatus;
+import com.mapbox.navigator.Navigator;
+
+import java.util.Date;
+
+class MapboxNavigator {
+
+  private final Navigator navigator;
+
+  MapboxNavigator(Navigator navigator) {
+    this.navigator = navigator;
+  }
+
+  void updateRoute(String routeJson) {
+    navigator.setDirections(routeJson);
+  }
+
+  NavigationStatus retrieveStatus(Date date) {
+    return navigator.getStatus(date);
+  }
+
+  void updateLocation(Location location) {
+    navigator.updateLocation(buildFixLocationFrom(location));
+  }
+
+  private FixLocation buildFixLocationFrom(Location rawLocation) {
+    Point rawPoint = Point.fromLngLat(rawLocation.getLongitude(), rawLocation.getLatitude());
```

Comment:
Should this be somewhere else? Seems more like a utils method since it doesn't rely on anything in the `Navigator`. Or we could have a wrapper class for `FixLocation` that does all of this in the constructor and/or a builder.

Reply:
This method has to do with `Navigator` because it can only accept `FixLocation`s - we can discuss further, but I don't think that necessitates a new util class